src/main/java/org/jenkinsci/plugins/registry/notification/webhook/dockerregistry/DockerRegistryWebHookPayload.java

```diff
@@ -46,9 +46,8 @@
     public DockerRegistryWebHookPayload(@Nonnull JSONObject data) {
         super();
         setData(data);
-        if (data != null) {
-            setJson(data.toString());
-        }
+        setJson(data.toString());
```

Comment:
why?

Reply:
The checkstyle hook was complaining as `data` is already marked as not-nullable :)